install_config/topics/using_gcp_load_balancer-proc.adoc

```diff
@@ -0,0 +1,113 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+= Using the GCP external load balancer as a service
+
+{product-title} can leverage the GCP load balancer capacity by exposing services
+externally using a `LoadBalancer` service. {product-title} creates the load
+balancer in GCP and creates the proper firewall rules.
+
+[NOTE]
+====
+Currently, port 10256 must be opened manually for service load balancers
+to work in GCP. See https://bugzilla.redhat.com/show_bug.cgi?id=1583833 for
+more information.
+====
+
+.Procedure
+
+. Create a `LoadBalancer` service:
++
+[source,json]
+----
+{
+  "kind": "Service",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "myservice"
+  },
+  "spec": {
+    "ports": [{
+      "port": 9876,
+      "targetPort": 9876
+    }],
+    "selector": {
+      "app": "example"
+    },
+    "type": "LoadBalancer"
+  }
+}
+----
+
+. Create a new application:
++
+[source,bash]
+----
+$ oc new-app openshift/hello-openshift
+----
+
+. Expose the load balancer service:
++
+----
+$ oc expose dc hello-openshift --name='hello-openshift-external' --type='LoadBalancer'
+----
+
+. To verify that the service has been created:
++
+----
+$ oc get svc
+NAME                       TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)                         AGE
+hello-openshift            ClusterIP      172.30.62.10     <none>          8080/TCP,8888/TCP               20m
+hello-openshift-external   LoadBalancer   172.30.147.214   35.230.97.224   8080:31521/TCP,8888:30843/TCP   19m
+----
++
```

Comment:
The `LoadBalancer` type and External IP fields indicate that the service is using GCP Load Balancers to expose that application.